tests/integration/test_splunk.py

```diff
@@ -0,0 +1,32 @@
+from tests.integration import basetest
+
+
+class TestCaseDeployWithSplunk(basetest.BaseTest):
+    def _deploy_app(self, mda_file):
+        super().setUp()
+        self.stage_container(
+            mda_file,
+            env_vars={
+                "SPLUNK_HOST": "test.splunkcloud.com",
+                "SPLUNK_TOKEN": "dummy_token",
+            },
+        )
+        self.start_container()
+
+    def _test_fluentbit_running(self, mda_file):
+        self._deploy_app(mda_file)
+        self.assert_app_running()
+
+        # check if Fluentbit is running
+        output = self.run_on_container("ps -ef| grep fluentbit")
+        assert output is not None
+        assert str(output).find("fluent-bit") >= 0
+
+    def _test_splunk_is_configured(self):
```

Comment:
Thanks, so this tests the scenario where the SPLUNK vars are enabled. Can you test the unhappy path too?

Reply:
Added negative test. Added regex support methods to base module